frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -120,6 +125,22 @@ const plugin: Plugin<ConsumedExtensions> = [
       template: yamlTemplates.getIn([models.FooBarModel, 'default']),
     },
   },
+  {
+    type: 'Dashboards/Overview/Health/URL',
+    properties: {
+      title: 'Foo system',
+      url: 'fooUrl',
+      healthHandler: getFooHealthState,
+    },
+  },
+  {
+    type: 'Dashboards/Overview/Health/Prometheus',
+    properties: {
+      title: 'Via prometheus',
```

Comment:
Nit-pick: `Bar system` for consistency :)

Reply:
`Bar system` it is ! :)